libs/ui-lib/lib/ocm/components/Routes.tsx

```diff
@@ -10,24 +17,38 @@ import { useFeatureDetection } from '../hooks/use-feature-detection';
 export const UILibRoutes = ({
   allEnabledFeatures,
   children,
+  history,
+  basename,
 }: {
   allEnabledFeatures: FeatureListType;
```

Comment:
since this is a required prop what should OCM-UI pass to it? or can it be made optional?

Reply:
I am not sure about that. Let's leave it as-is both here and in UHC-Portal.

I can discuss this with @ammont82 when she's back from PTO.